apps/archive/archive_media.py

```diff
@@ -102,3 +114,24 @@ def get_file_from_document(self, doc):
             return content, content_type, decode_metadata(metadata)
 
         return file, file.content_type, file.metadata
+
+    def upload_file_to_video_server(self, doc):
+        """
+        Upload file to video server and create rendition for it
+        :param doc: info of file
+        :return:
+        """
+        file = doc.get('media')
+        # upload video to video server
+        res = self.videoEditor.post(doc.get('media'))
+        doc['media'] = res['_id']
+        metadata = res.get('metadata')
+        # create renditions
+        rend = {
+            'href': res.get('url'),
```

Comment:
We should not use direct reference to a video server here, because a browser will not have a direct access to it. 
All requests to `raw` resources (video, preview thumbnail and timeline thumbnails) should go through superdesk.
As an option we can do some nginx magic for this `raw` urls. 

UPD: This will be solved with nginx config, so let's keep it as it is.